commit/trk2dictionary/trk2dictionary.pyx

```diff
@@ -35,6 +35,8 @@ cpdef run( filename_tractogram, path_out, TCK_ref_image = None, filename_peaks =
     ----------
     filename_tractogram : string
         Path to the .trk or .tck file containing the tractogram to convert.
+        
+    filename_trk : DEPRECATED
```

Comment:
Should be: "filename_trk : DEPRECATED. Use filename_tractogram instead."

(or something similar)